CommonApi/src/main/java/mezz/jei/api/gui/IRecipeLayoutDrawable.java

```diff
@@ -87,6 +89,12 @@ default Optional<ItemStack> getItemStackUnderMouse(int mouseX, int mouseY) {
 	 */
 	IRecipeCategory<R> getRecipeCategory();
 
+	/**
+	 * Get the decorators for the recipe category of this recipe layout.
+	 * @since 15.1.0
+	 */
+	Collection<IRecipeCategoryDecorator<R>> getRecipeCategoryDecorators();
```

Comment:
I think we can remove this and keep it as an internal detail

Reply:
One thing to keep in mind is that anything added to the API will stay around for a long time. Any changes we make will need to go through a version-long deprecation cycle. I try to add the minimum required for a new feature, we can decide to add more later if needed.